podpac/datalib/smap.py

```diff
@@ -133,8 +175,32 @@ def np2smap_date(date):
               }
 )
 
-SMAP_INCOMPLETE_SOURCE_COORDINATES = ['SPL2SMAP_S.001']
-SMAP_BASE_URL = 'https://n5eil01u.ecs.nsidc.org/opendap/hyrax/SMAP'
+SMAP_INCOMPLETE_SOURCE_COORDINATES = ['SPL2SMAP_S']
+SMAP_IRREGULAR_COORDINATES = ['SPL2SMAP_S']
+
+# Discover SMAP OpenDAP url from podpac s3 server
```

Comment:
In what follows, I try to retrieve the SMAP openDAP url from our S3 bucket, the file on disk, falling back on the hard-coded value if all of that fails. 

***NOTE*** This url should eventually come from the NASA CMR.